src/test/java/hw2/TestBase.java

```diff
@@ -0,0 +1,82 @@
+package hw2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.asserts.SoftAssert;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TestBase {
```

Comment:
Class with "Base" in name should has abstract modifier according to java convention

Reply:
done